low-level/files/iec_mapping/sspl

```diff
@@ -0,0 +1,10 @@
+0050010001,Ipmitool,Ipmitool command execution error
```

Comment:
Keep actual commands in messages (maintaining case), like ipmitool, hdparam, smartctl(remove smartmontools as its pkg), udisks (py pkg?)

Reply:
Done.